x-pack/plugins/inference/server/routes/chat_complete.ts

```diff
@@ -88,6 +88,12 @@ export function registerChatCompleteRoute({
   router.post(
     {
       path: '/internal/inference/chat_complete',
+      security: {
+        authz: {
+          enabled: false,
+          reason: 'This route is opted out from authorization',
+        },
+      },
```

Comment:
@dgieselaar I don't think we intended to opt-out of authorization here? Don't we want to require a minimum set of privileges?

Reply:
That's true, but atm we don't know what set of roles or permission swe should restraint to (virtual any plugin can call those APIs from the inference public contract). Plus, RBAC regarding inference APIs could be lower level than the endpoints.

The PR is just adapting the code to the new route rbac format, basically

Reply:
Can we model privileges based on the features that currently call this endpoint? For example, could this route require any of the privileges that grant access to the AI Assistants?